packages/dso-toolkit/src/legacy/legacy-panel.scss

```diff
@@ -226,8 +226,11 @@ $panel-danger-heading-bg: states.$danger-bg !default;
   @include utilities.clearfix();
   @include rich-content.children();
 
-  padding: $panel-body-padding;
-  padding-block-end: 0;
+  /* stylelint-disable no-duplicate-selectors -- needed to fix sass deprecation warnings: #2724 */
```

Comment:
Comments met Sass comments `//` plaatsen zodat deze niet in de output `dso.css` komen.

Je hebt het overal goed gedaan, en zelfs wat oude `/* */` omgezet naar `//`, alleen deze nog.